src/model/operation/transform.js

```diff
@@ -52,12 +53,11 @@ import compareArrays from '@ckeditor/ckeditor5-utils/src/comparearrays';
  * `a` by `b` and also `b` by `a`. In both transformations the same operation has to be the important one. If we assume
  * that first or the second passed operation is always more important we won't be able to solve this case.
  *
- * @external module:engine/model/model.operation
+ * @protected
```

Comment:
I'll remove this because it creates a precedence that we have a protected export. We should rather invest in marking internal modules all across the code.